### ISSUE TITLE
Classify Fortasyn mice - stroke or not

### ISSUE THREAD

**Issue by WolfByttner** [traja-team/traja#50]
Fortasyn is a dataset of mice that went through an induced stroke. There are 24 mice in the dataset. By looking at the trajectories, how accurately can we determine if a mouse has suffered a stroke or not?

Exit criteria:
* A classifier has been implemented in a Jupyter notebook.
* The classifier successfully classifies Fortasyn mice, or we have conclusively determined that classification is not possible.
* The notebook is added to traja-research.

**Comment by MaddyThakker** [comment]
### Dataset - 
The dataset used for this experiment is [E1_E2.csv](http://justinshenk.com/share/E1_E2.csv). 
This dataset has information for 2 mice with 599,581 datapoints with 10 seconds 

### Experiment - 

1. Entire dataset (599581 datapoints) used - The model seems to learn anything and the accuracy is always around 0.5 (same as random guess) even with different hyperparameter settings and trained for over 200 epochs. 

2. Day-wise dataset - The dataset was divided into buckets with each bucket having dataset for each day ( ~ 17,500 datapoints for each day) further divided into train-val-test sets. 

Some days are easier to learn as compared to the others. Here are some of the results - 

- Day 1:
<img width="589" alt="day-1" src="https://user-images.githubusercontent.com/23390805/109411935-4a7d9c00-79cb-11eb-817d-078193606837.png">

- Day 2:
<img width="583" alt="day-2" src="https://user-images.githubusercontent.com/23390805/109411944-4f425000-79cb-11eb-9c6f-e89875186bcb.png">

- Day 3:
<img width="616" alt="day-3" src="https://user-images.githubusercontent.com/23390805/109411945-50737d00-79cb-11eb-9e83-4c1238749bb7.png">

- Day 1,2,3:
<img width="617" alt="day-1,2,3" src="https://user-images.githubusercontent.com/23390805/109411942-4d788c80-79cb-11eb-955a-0f7c41844620.png">

- Resconstruction Day4:
![vis-day4](https://user-images.githubusercontent.com/23390805/109411946-510c1380-79cb-11eb-9046-6ebd23585a93.png)

### Colab notebook  - 
[https://colab.research.google.com/drive/1XW2Do7IqeQZGZjyJDWLlw0WgXpJmsSBC?usp=sharing](https://colab.research.google.com/drive/1XW2Do7IqeQZGZjyJDWLlw0WgXpJmsSBC?usp=sharing)

### Conclusion - 
The dataset varies quite a lot over different days and is incredibly hard to learn a general classifier over the entire dataset. If divided by days, the learning seems to get better on some days. On other days, it is as good as random guess.
